server/util.go

```diff
@@ -242,6 +243,18 @@ func interceptConfigs(rootViper *viper.Viper) (*tmcfg.Config, error) {
 		return nil, fmt.Errorf("failed to merge configuration: %w", err)
 	}
 
+	// Adding default ClientConfig and writing it into "client.toml"
+	cliCfgFilePath := filepath.Join(configPath, "client.toml")
+	if _, err := os.Stat(cliCfgFilePath); os.IsNotExist(err) {
```

Comment:
I am not quite sure if I have to check whether os.IsNotExist(err) holds true. Essentially, cliCfgFilePath does not exist before clicfg.WriteConfigFile(cliCfgFilePath, cliConfig, configTemplate) is called on the line 255. Please advice.